torch_ttnn/passes/to_tt_pass.py

```diff
@@ -368,6 +368,17 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                     new_node,
                     delete_user_cb=lambda node: node != new_node,
                 )
+            if node.target == torch.ops.aten.transpose.int:
+                dim0 = args[1]
+                dim1 = args[2]
+                rank = len(node.meta["val"].size())
+                permutation = list(range(rank))
+                permutation[dim0], permutation[dim1] = permutation[dim1], permutation[dim0]
+                new_node = g.call_function(ttnn.permute, args=(args[0], permutation))
+                node.replace_all_uses_with(
+                    new_node,
+                    delete_user_cb=lambda node: node != new_node,
```

Comment:
what does this do?

Reply:
This replaces the users of existing torch node attaching to the users list of newly created ttnn node. Then the torch node can be deleted safely.